services/src/api/model/datatypes.rs

```diff
@@ -1567,7 +1586,7 @@ pub enum RasterColorizer {
 impl RasterColorizer {
     pub fn band_selection(&self) -> BandSelection {
         match self {
-            RasterColorizer::SingleBand { band, .. } => BandSelection::new_single(*band),
+            RasterColorizer::SingleBand { band, .. } => BandSelection(vec![*band as usize]),
```

Comment:
why did you remove the constructor?

Reply:
I didn't remove the constructur but the type is now from `api` and not from `datatype`. The former one just doesn't have this method because it's only created by deserialization of user data and not from code.